src/grate_asm/fragment_asm.tab.c

```diff
@@ -0,0 +1,3430 @@
+/* A Bison parser, made by GNU Bison 3.0.4.  */
```

Comment:
Did you really mean to commit this file? Shouldn't this be produced by the build-system instead?

Reply:
I wanted to avoid depending on flex / bison as you supposed rightly. I'm not familiar with how mesa is dealing with that and I don't quite understand what "generate these files when generating release tarballs" means, could you please elaborate? To generate files we would need the grate_asm in any form anyway, so I'm a bit lost here.

But maybe dependency on flex / bison isn't really a burden? Another option could be to factor out grate-asm into a standalone repo (new grate-driver's github) and add git submodule (or subtree?) directory with the grate-asm to opentegra. That way when 'make' is running, it will pull grate-asm git and build the assembler, so we will have the assembler shared and won't have a library-dependency. For example QEMU is doing it that way and I'm finding it very-very convenient.

Reply:
Okay, I don't have a good opnion about what to do with it now. Maybe grate-asm factoring out and making it a 'usual' build-dep is fine too.

You prefer to not to have generated files in opentegra and you'd want the shared code. So the options are:

1. Factor out grate-asm to a standalone package
2. Factor out grate-asm to git-submodule
3. Neglect code sharing and have asm in YACC/FLEX form in opentegra
4. ?

Reply:
Let's go with 3. for now and consider 4 once we have something other than opentegra that uses grate-asm.